CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [4.3.0] - 2021-07-29
```

Comment:
I think you should add the changes to the "Unreleased" section and @traversaro will move it to a specific version when he will release it. But I will let him confirm on this

Reply:
Yes, @francesco-romano  is correct, just deleting that header should be enough.
```suggestion
```

We are going to release quite soon, but I can't promise to do a release yesterday without breaking the rules of space and time. : ) 
See https://keepachangelog.com/en/1.0.0/#effort for more details on the idea of the `[Unreleased]` section.